### PR TITLE
added functionality to add new projects as templates and messed with …

### PR DESCRIPTION
**Pull request by loganlsr**
…styling

### CODE REVIEW

**Comment by loganlsr** [comment]
Got it working after about 2 hours and spent about 2 more messing around with CSS.

I forgot to add the thing for the publish date where you do /60/60/24/1000 to get the number of days. I need to remember that for the future.  My CSS is kind of a mess under the hood, so I need to work on trying to condense and clean that up if possible.

File: JS/app.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+var projects = [];
+
+function Project(opts) {
+  this.author = opts.author;
+  this.title = opts.title;
+  this.category = opts.category;
+  this.projectUrl = opts.projectUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
+  console.log('successfully calling construtor');
```
Comment (nhuntwalker) [inline JS/app.js:12]: This console.log should've been removed

File: JS/app.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+}
+
+Project.prototype.toHtml = function(){
+  var $newProject = $('project.template').clone();
+  console.log('just cloned project template');
```
Comment (nhuntwalker) [inline JS/app.js:17]: Remove all console.logs when you submit

File: JS/projectObjects.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+'use strict';
+
+var projectData = [
```
Comment (nhuntwalker) [inline JS/projectObjects.js:3]: In an ideal world, you would use something that had more than one entry.

**Pull request merged**